microsite/src/pages/plugins/_pluginCard.tsx

```diff
@@ -9,14 +9,14 @@ export interface IPluginData {
   category: string;
   description: string;
   documentation: string;
-  iconUrl: string;
+  iconUrl?: string;
   title: string;
   addedDate: string;
   isNew: boolean;
   order?: number;
 }
 
-const defaultIconUrl = 'img/logo-gradient-on-dark.svg';
+const defaultIconUrl = '/img/logo-gradient-on-dark.svg';
```

Comment:
might be worth opening a separate PR for this fix as we could ship it straight away :+1:

Reply:
Sure, give me a sec ;)

Reply:
Hey @Rugvip, see #27920